Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "parquet-key-management"
+version = "0.1.0"
+description = "Implements the Parquet Key Management Tools API in Rust to enable integration with a Key Management Server when using Parquet modular encryption."
+edition = "2024"
```

Comment:
should we align with the edition that `parquet` is targeting? (2021)

Reply:
Different editions need to interoperate across crates, but we probably do want to align our minimum supported rust version. The 2024 edition is only available since Rust 1.85, and arrow-rs has a MSRV of 1.81, so I'll change this.